### ISSUE TITLE
 [VKSdk authorize:scope] doesn't work

### ISSUE THREAD

**Issue by Joicker** [VKCOM/vk-ios-sdk#538]
use the  [VKSdk authorize:scope] is nothing happen ，where should i check it

**Comment by oboupo** [comment]
do you set up delegates?
`VKSdk *sdkInstance = [VKSdk initializeWithAppId:YOUR_APP_ID];
[sdkInstance registerDelegate:delegate];
[sdkInstance setUiDelegate:uiDelegate];`

**Comment by Joicker** [comment]
yep,but the [VKSdk wakeUpSession:SCOPE completeBlock:^(VKAuthorizationState state, NSError *error) is not running ,and i use this one [[VKSdk initializeWithAppId:APP_ID] registerDelegate:delegate];

**Comment by oboupo** [comment]
Do you set UI delegate?

**Comment by Joicker** [comment]
yes i did

**Comment by oboupo** [comment]
Please show you full code where you initialize SDK and call authorise method

**Comment by Joicker** [comment]
![image](https://user-images.githubusercontent.com/34057074/43296361-7977b088-917d-11e8-952c-ab6c28a9f091.png)
![image](https://user-images.githubusercontent.com/34057074/43296387-a483a0fc-917d-11e8-8ebd-8d96d21c77b4.png)

**Comment by oboupo** [comment]
Are you sure that `int vklogin(lua_state *L)` called after your method on first screenshoot?

**Comment by Joicker** [comment]
I'm sure ，the first screenshoot i was set in the -(BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions

**Comment by Joicker** [comment]
still here？
